packages/core/src/html/NodeProcessor.js

```diff
@@ -264,7 +264,9 @@ class NodeProcessor {
     if (linkProcessor.hasTagLink(node)) {
       linkProcessor.convertRelativeLinks(node, context.cwf, this.config.rootPath, this.config.baseUrl);
       linkProcessor.convertMdAndMbdExtToHtmlExt(node);
-      if (this.config.intrasiteLinkValidation.enabled) {
+      const isSourceFile = context.callStack.length === 0
```

Comment:
on the same lines, this may disable link checking entirely for files that are only `<include />`-ed (not used as a page)